data/tools/converters.yml

```diff
@@ -292,6 +294,7 @@
         comment: |
             Convert DICOM and NIFTI data to the ASL-BIDS format.
         data_types:
+        -   MRI
         -   ASL
```

Comment:
OK this makes me realize that our terminology is not clear here.

MRI in the BIDS world is a datatype, it's a modality.

ASL does not exist as such in BIDS: it is the `perf` datatype under the `MRI` modality.

EEG is the EEG datatype under the EEG modality.

I don't think we should fix this in this PR, but it'd be good to address this later.

Reply:
Agree, we don't want to fall to far into minutia (definitely not our mandate ;)). Save that for another PR.